tutor/src/components/buttons/save-practice.js

```diff
@@ -81,7 +82,7 @@ const SavePracticeButton = observer(({
             practiceQuestion.destroy();
         }
         else {
-            practiceQuestions.create(taskStep.tasked_id);
+            runInAction(() => practiceQuestions.create(taskStep.tasked_id));
```

Comment:
Wouldn't destroy above also need this runInAction()?

Reply:
I almost added it too but then saw that `destroy` here is a wrapper that calls the actual `map.delete` (with `runInAction`) after doing the network request.